soh/soh/Enhancements/game-interactor/GameInteractor_Actions.cpp

```diff
@@ -0,0 +1,208 @@
+#include "GameInteractor.h"
+#include <libultraship/bridge.h>
+
+extern "C" {
+#include "variables.h"
+#include "macros.h"
+#include "functions.h"
+extern PlayState* gPlayState;
+}
+
+#include "overlays/actors/ovl_En_Niw/z_en_niw.h"
+
+void GameInteractor::Actions::AddOrRemoveHealthContainers(int32_t amount) {
+    gSaveContext.healthCapacity += amount * 0x10;
```

Comment:
mentioned this in my comments on `debugconsole`, but it'd be nice to move those checks here instead